examples/components/RadarDemo.vue

```diff
@@ -45,28 +47,29 @@ export default {
     vm.wedge = {
       x: vm.configKonva.width / 2,
       y: vm.configKonva.height / 2,
-      radius: 100,
+      radius: 160,
       angle: 60,
       direction: 90,
       draggable: true,  // todo: after dragging, changing direction and angle makes radar to go back initial position
       fillRadialGradientStartPoint: 0,
       fillRadialGradientStartRadius: 0,
       fillRadialGradientEndPoint: 0,
-      fillRadialGradientEndRadius: 100,
+      fillRadialGradientEndRadius: 160,
       fillRadialGradientColorStops: [0, 'red', 0.3, 'blue', 0.5, 'rgba(0,0,255,0.6)', 1, 'rgba(255,255,255,0)'],
       opacity: 0.8,
       stroke: 'transparent',
       strokeWidth: 1,
       scaleX: 1,
       scaleY: 1,
-      startScale: 1,
-      handler: vm.wedgeHangler   // todo: draw handler for radar to rotate (to change direction)
+      startScale: 1
     }
-    vm.wedgeHangler = {
+
+    vm.handler = { // todo: draw handler for radar to rotate (to change direction)
       radius: 10,
-      angle: 60,
-      opacity: 0.8,
-      strokeWidth: 1
+      strokeWidth: 2,
+      stroke: 'rgba(0,0,255,0.5)',
+      fill: 'rgba(255,255,255,1)',
+      draggable: true // this should be always true by default
```

Comment:
Because you want to change opacity with `onHandlerDragStart()` and `onHandlerDragEnd()`, then you should define opacity here.
With Vue, property must be present on initial state if you want it to be 'reactive'. More info here : https://v1.vuejs.org/guide/reactivity.html

Reply:
As you can see, with current code, visual opacity is not affected by code because reactivity is not set on that property.

Reply:
Yeah it is not affected, let me try again